bindings.txt

```diff
@@ -0,0 +1,6 @@
+--in .windows/winmd/
+# TODO: Maybe flatten?
```

Comment:
Is this still an open task or is just a leftover?

Reply:
Hmm no `--config flatten` generates really useless bindings because it duplicates everything from the `windows`/`windows-core` crates, giving incompatible types. We'll have to accept the namespace nesting.

Reply:
Please open a new issue if there's something about code gen that I can improve.

Reply:
@kennykerr I'm not sure what such an improvement would look like. I think it's fine to have the namespace nesting.